src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -10,6 +10,8 @@
       "major": "Computer Science",
       "intake": "2023",
       "tags": [
+        "TA",
+        "Smart",
```

Comment:
Should we modify the typicalPersonAddressBook?

Reply:
I think we shouldn't , this one is applicable only if the address book is empty